src/inputs/ObjectSearchCreate.tsx

```diff
@@ -12,7 +12,6 @@ import { IFieldConfig, IFieldConfigObjectSearchCreate } from '../';
 interface IProps {
   fieldConfig: IFieldConfig;
   form: any;
-  isVisible?: any;
   label: string;
```

Comment:
Why have a label prop if only the label in `fieldConfig` is used?

Reply:
This is a great question, I'm actually gonna look into this a bit more...

Reply:
Answer: There shouldn't be, label is never passed.